locale/update_po_files.sh

```diff
@@ -2,7 +2,7 @@
 # Run this script with locale as the current directory
 set -o errexit
 echo ";; Recreating audacity.pot using .h, .cpp and .mm files"
-for path in ../modules/mod-* ../libraries/lib-* ../include ../src ; do
+for path in ../modules/mod-* ../libraries/lib-* ../include ../src ../crashreports ; do
```

Comment:
At least one more thing before merging:

Make sure new translated strings and i18n-hint comments are exactly as in UnusedStrings.h, then delete them from there.   Commit that.

Run locale/update_po_files.sh to verify that the strings are not removed from Audacity.pot.  (The lines may move.). Do not commit that.

Reply:
Done